django/core/servers/basehttp.py

```diff
@@ -51,8 +51,8 @@ def get_internal_wsgi_application():
 
 
 def is_broken_pipe_error():
-    exc_type, exc_value = sys.exc_info()[:2]
-    return issubclass(exc_type, socket.error) and exc_value.args[0] == 32
+    exc_type, _exc_value, _traceback = sys.exc_info()
+    return issubclass(exc_type, BrokenPipeError)
```

Comment:
May as well flatten to one line and use `sys.exc_info()[0]`.

Reply:
It is minor and probably comes down to personal preference, but I find the current form slightly more readable and less dense. It shows exactly what `sys.exc_info()` returns with descriptive variables names (a new contributor might not be familiar with this function). It is immediately clear that is is a 3-tuple and not something else like a list. This form also desn't overload a single line with so many operations.

But, again, I recognize this is personal preference so this line change is required for it to be agreeable, I'll change it.